src/bun.js/webcore/response.zig

```diff
@@ -830,7 +1022,10 @@ pub const Fetch = struct {
             const allocator = bun.default_allocator;
             var response = allocator.create(Response) catch unreachable;
             response.* = this.toResponse(allocator);
-            return Response.makeMaybePooled(@as(js.JSContextRef, @ptrCast(this.global_this)), response);
+            const response_js = Response.makeMaybePooled(@as(js.JSContextRef, @ptrCast(this.global_this)), response);
```

Comment:
the `@ptrCast` here does nothing btw

Reply:
fixed on https://github.com/oven-sh/bun/pull/4127/commits/ef6989f4118ee842adf88f83c9d2b3ba64f04f0d